.github/workflows/samcli-direct.yaml

```diff
@@ -15,6 +18,7 @@ permissions:
 
 jobs:
   samcli-direct-test:
+    if: github.event_name == 'schedule' || github.event_name == 'workflow_dispatch' || github.event.pull_request.merged == true
```

Comment:
Why do we need workflow_dispatch? is this workflow being triggered by some other workflow?